validation/src/test/scala/hmda/validation/rules/lar/validity/V540Spec.scala

```diff
@@ -0,0 +1,43 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V540Spec extends LarEditCheckSpec with BadValueUtils {
+
+  val badActionTakenTypes = List(2, 3, 4, 5, 7, 8)
+  val validActionTakenGen: Gen[Int] =
```

Comment:
I suggest not using "Types" in your variable naming moving forward, unless you are doing a `type` alias in `Scala`. I would use something like `badActionTakenValues`